pkg/ngrokapidriver/driver.go

```diff
@@ -90,7 +89,7 @@ func (napi ngrokAPIDriver) CreateEdge(ctx context.Context, edgeSummary *Edge) (*
 	}
 
 	newEdge, err := napi.edges.Create(ctx, &ngrok.HTTPSEdgeCreate{
-		Hostports:   &[]string{edgeSummary.Hostport},
```

Comment:
This was correct when we were using `ngrok-api-go/v4`. The latest release introduced some breaking changes to be more idiomatic since slices are pointers to underlying arrays. Its called out in the [v5 release notes](https://github.com/ngrok/ngrok-api-go/releases/tag/v5.0.0)